### PR TITLE
fix(kubernetes): new ingressRoutes and env variables

### LINKED ISSUE #748: Fix Kubernetes Issues

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #748 

### What does this PR do?
new ingressRoutes and env variables

### CODE REVIEW

File: .github/workflows/push-dev-environment.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
           NAMESPACE: lifebank
           STAGE: dev
           APP_NAME: lifebank
-          INGRESS_HOST: lifebank.io
+          INGRESS_HOST: lifebank.eosio.cr
```
Comment (Edgar097) [inline .github/workflows/push-dev-environment.yml:27]: Is this change ok?  or there should be lifebank.io and lifebank.eosio.cr separated?

Comment (xavier506) [reply .github/workflows/push-dev-environment.yml:27]: lifebank.io

File: kubernetes/ingress.yaml [hunk 2]
```diff
@@ -53,11 +53,45 @@ spec:
...
+kind: IngressRoute
+metadata:
+  name: https
+  namespace: lifebank
+spec:
+  entryPoints:
+    - websecure
+  routes:
+    - kind: Rule
+      match: Host(`${INGRESS_GRAPHQL_HOST}`)
```
Comment (xavier506) [inline kubernetes/ingress.yaml:91]: This line should be  `match: Host(`lifebank.eosio.cr`)`

File: kubernetes/ingress.yaml [hunk 3]
```diff
@@ -53,11 +53,45 @@ spec:
...
+kind: IngressRoute
+metadata:
+  name: http
+  namespace: lifebank
+spec:
+  entryPoints:
+    - web
+  routes:
+    - kind: Rule
+      match: Host(`${INGRESS_GRAPHQL_HOST}`)
```
Comment (xavier506) [inline kubernetes/ingress.yaml:75]: `match: Host("lifebank.eosio.cr")`

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by kuronosec — COMMENTED** [review]
LGTM

**Review by kuronosec — COMMENTED** [review]
LGTM

**Pull request merged**